keanu-project/build.gradle

```diff
@@ -39,6 +39,7 @@ dependencies {
     testCompile 'org.jfree:jfreechart:1.0.19'
     testCompile group: 'org.mockito', name: 'mockito-core', version: '2.15.0'
     testCompile 'org.hamcrest:hamcrest-library:1.3'
+    testCompile 'ca.umontreal.iro.simul:ssj:3.2.1'
```

Comment:
This library hasn't been touched in 6 years... perhaps we should use this one instead which is actively developed: https://github.com/stanfordnlp/CoreNLP/blob/master/src/edu/stanford/nlp/stats/Dirichlet.java

Reply:
As discussed - that one doesn't look ideal either (some of the functions are just stubs).  The Dirichlet implementation seems sound (it agrees with Georges) so are we ok to stick with this?

Reply:
agreed